src/fb.js

```diff
@@ -53,24 +53,34 @@ var {version} = require('../package.json'),
 		return JSON.parse(header['x-page-usage']);
 	},
 	buildRateLimitObjectFromJson = function(rateLimit) {
-		return Object.assign(Object.create(null), {
+		return {
 			callCount: rateLimit['call_count'],
 			totalTime: rateLimit['total_time'],
 			totalCPUTime: rateLimit['total_cputime']
-		});
+		};
 	},
 	stringifyParams = function(params) {
-		var data = {};
+		let data = {};
+		const keys = Object.keys(params);
 
-		for ( let key in params ) {
+		keys.forEach(key => {
```

Comment:
If you're refactoring this to use es6+ I'd prefer for..of Object.entries over the ES5 `forEach`.